pkgs/by-name/co/cortex-command-community-project/package.nix

```
@@ -0,0 +1,79 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  flac,
+  lib,
+  libgcc,
+  libGL,
+  libpng,
+  libpulseaudio,
+  lz4,
+  luajit,
+  meson,
+  minizip,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  SDL2,
+  SDL2_image,
+  tbb,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cortex-command-community-project";
+  version = "6.2.2";
+
+  src = fetchFromGitHub {
+    owner = "cortex-command-community";
+    repo = "Cortex-Command-Community-Project";
+    rev = "v${version}";
+    sha256 = "sha256-srbV6Nh+ecyV0dkY835vhzpMSzmnvANym453L72cmGI=";
+  };
+
+  patches = [
+    ./runner.patch
+  ];
+
+  postFixup = ''
+    patchelf --set-rpath ${lib.makeLibraryPath [ libpulseaudio ]} $out/lib/CortexCommand/libfmod.so*
+  '';
+
+  nativeBuildInputs = [
+    libgcc
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    SDL2
+    flac
+    libpng
+    libpulseaudio
+    libGL
+    lz4
+    luajit
+    minizip
+    SDL2_image
+    tbb
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Cortex Command Community Project Source";
+    longDescription = ''
+      The Cortex Command Community Project is Free/Libre and Open Source under GNU AGPL v3.
+      This is a community-driven effort to continue the development of Cortex Command.
+    '';
+    homepage = "https://cortex-command-community.github.io/";
+    changelog = "https://github.com/cortex-command-community/Cortex-Command-Community-Project/releases";
```

```suggestion
    changelog = "https://github.com/cortex-command-community/Cortex-Command-Community-Project/blob/v${finalAttrs.version}/CHANGELOG.md";
```
Seems like upstream points people to their `CHANGELOG.md` file for the full changelog.